### ISSUE TITLE
Migrate legacy Andisol ids

### ISSUE THREAD

**Issue by FewerFlaws** [TerraformersMC/Terrestria#227]
Andisol dirt, grass blocks, and podzol use "basalt_dirt" "basalt_grass_block" and "basalt_podzol" within the namespace.  This is inconsistent because the grass path and farmland blocks use "andisol_grass_path" and "andisol_farmland."

I imagine you already know about this and decided not to correct it.  I'm just pointing it out in case it's an oversight.  I noticed it while working on compatibility between Presence Footsteps and Terrestria =]

**Comment by Prospector** [comment]
Yeah, this cannot be reasonably changed without breaking worlds. Perhaps we can fix it in the transition to 1.17 by using a datafixer, but unfortunately I don't believe we can get datafixers to run easily between mod version changes.

Luckily, ultimately it doesn't really matter what the ids are.

**Comment by FewerFlaws** [comment]
New development on this.  REI displays "andisol_grass_path" in the interface, but F3 debug view reveals that a placed block uses "basalt_grass_path"
![2020-12-28_22 56 55](https://user-images.githubusercontent.com/75643699/103264721-6a0f2e80-4960-11eb-91e6-798496c52ca5.png)
![2020-12-28_22 57 03](https://user-images.githubusercontent.com/75643699/103264728-6c718880-4960-11eb-9bb8-b545e5a70f01.png)

**Comment by Prospector** [comment]
Yeah, the item was added later after the rename.

**Comment by FewerFlaws** [comment]
Surely the tooltip issue can be fixed, right?  The name shown in the extended F3+H tooltip should be the actual namespace name so that mod authors can see the block and reference it properly.  Fixing that shouldn't break anything.

**Comment by FewerFlaws** [comment]
Oh, I didn't describe the issue with the grass path properly.  Check out that first screenshot to see the issue.  The Andisol grass path's extended tooltip is displaying the wrong namespace block ID.

**Comment by Prospector** [comment]
No it's not. Item IDs are different from block IDs.

**Comment by FewerFlaws** [comment]
Oh, I understand.  Thanks =]
